queues_test.go

```diff
@@ -75,13 +75,13 @@ func testQueues(t *testing.T, setup NewStorageFunc, tearDown func()) {
 			var queueName = random.String("queue-", 10)
 			now := clock.Now().UTC()
 			require.NoError(t, c.QueuesCreate(ctx, &pb.QueueInfo{
```

Comment:
Just a random note: was `queues_test.go` and `queue_test.go` meant to be combined to one file?

Reply:
yes, one is testing `queues` and the other is testing the operation of a `queue`. I'm really trying to avoid overloading the usage of the word `queue` so... maybe we should call it something else?